docs/ibis-for-dplyr-users.ipynb

```diff
@@ -0,0 +1,777 @@
+{
```

Comment:
<p>Should one of the <code>summarize</code>s have an <code>s</code>?    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6050/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>Yes! Good eye, thanks.</p>